boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -333,16 +333,34 @@ PcMemGetBiosMemoryMap(PFREELDR_MEMORY_DESCRIPTOR MemoryMap, ULONG MaxMemoryMapSi
             break;
         }
 
-        if (((PBIOS_MEMORY_MAP)BIOSCALLBUFFER)->ExtendedAttributes.Enabled_Reserved == 0)
+        if (((PBIOS_MEMORY_MAP)BIOSCALLBUFFER)->ExtendedAttributes.Enabled_Reserved != 1)
         {
-            WARN("Discard disabled/invalid entry. (would-be-PcBiosMapCount = %lu)\n",
+            WARN("Discarding disabled/invalid entry. (would-be-PcBiosMapCount = %lu)\n",
                  PcBiosMapCount);
             /* This unlikely case was correct between ACPI 3.0 and 4.0, so assume all is fine.
              * Unless we would be ready to drop ACPI 3.0 compatibility.
              */
             goto nextRange;
         }
 
+        if (((PBIOS_MEMORY_MAP)BIOSCALLBUFFER)->ExtendedAttributes.NonVolatile_Deprecated_Reserved != 0)
+        {
+            WARN("Ignoring deprecated/invalid NonVolatile flag. (PcBiosMapCount = %lu)\n",
```

Comment:
I don't think we need more dprints that don't add any value

Reply:
I agree they are of low value, yet I would like to have this code as a reminder.
Would adding an "#ifdef acpi_future", so it is not compiled, be acceptable?

Reply:
Just use a comment instead of a DPRINT.

Reply:
If `WARN()` is commented out, then this code doesn't do anything anymore,
hence the suggestion to use a global `#ifdef` (or `/* ... */`), around this code as is.